src/config.rs

```diff
@@ -356,7 +355,6 @@ pub struct DbConfig {
     pub writable_file_max_buffer_size: ReadableSize,
     pub use_direct_io_for_flush_and_compaction: bool,
     pub enable_pipelined_write: bool,
-    pub backup_dir: String,
```

Comment:
Is it backward compatible? If `backup_dir` is configured, can the configuration file be parsed successfully?

Reply:
Yes, unspecified configs will be ignored.